testing/libcli/APKBUILD

```diff
@@ -0,0 +1,28 @@
+# Contributor: arch3y <arch3y@riseup.net>
+# Maintainer: arch3y <arch3y@riseup.net>
+pkgname=libcli
+pkgver=1.9.7
+pkgrel=2
+pkgdesc="A library for including a Cisco-like command-line interface"
+url="https://github.com/dparrish/libcli"
+arch="all"
+options="!check"
```

Comment:
Could you add a comment indicating why `check` will not work, e.g. `# upstream has no test suite`?